ApollosProject/apollos-prototype
# Add Prayer Icon

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds a "pray" icon.
![Screenshot 2019-04-16 08 53 29](https://user-images.githubusercontent.com/3229463/56211289-8ef52e00-6025-11e9-944d-764b6b907558.png)
![Screenshot 2019-04-16 09 54 32](https://user-images.githubusercontent.com/3229463/56215415-a801dd00-602d-11e9-9ca7-ff106d7da561.png)

### What design trade-offs/decisions were made?
None

### How do I test this PR?
Run the app locally and change one of the icons to "pray". The icon should be the praying guy.

### What automated tests did you write?
None

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
